phalcon/mvc/model/manager.zep

```diff
@@ -1805,4 +1811,888 @@ class Manager implements ManagerInterface, InjectionAwareInterface, EventsAwareI
 		phalcon_orm_destroy_cache();
 		Query::clean();
 	}
+
+
+
+
+
+
+
+
```

Comment:
No need for speed, it will be at least few months until we create 4.0.x repo i guess, and few years to release :D